share/doc/homebrew/Acceptable-Formulae.md

```diff
@@ -104,12 +105,9 @@ running a package manager.
 
 ### Stuff that builds a .app
 Don’t make your formula build an `.app` (native OS X Application); we
-don’t want those things in Homebrew. Make it build a command line tool
-or a library. However, we have a few exceptions to that, e.g. when the
-App is just additional to CLI or if the GUI-application is non-native
-for OS X and/or hard to get in binary elsewhere (example: fontforge).
-Check out the [homebrew-cask](https://github.com/caskroom/homebrew-cask)
-project if you’d like to brew native OS X Applications.
+don’t want those things in Homebrew (but
+[homebrew-cask](https://github.com/caskroom/homebrew-cask) does).
```

Comment:
I wonder if this should be “Homebrew Cask” instead of “homebrew-cask” as the link text, as the latter is very close to how we otherwise refer to our taps (except for the hyphen instead of the slash) and could cause more confusion. Thoughts?